src/senaite/core/schema/durationfield.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of SENAITE.CORE.
+#
+# SENAITE.CORE is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, version 2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Copyright 2018-2024 by it's authors.
+# Some rights reserved, see README and LICENSE.
+
+from senaite.core.schema.fields import BaseField
+from senaite.core.schema.interfaces import IDurationField
+from zope.interface import implementer
+from zope.schema import Timedelta
+
+
+@implementer(IDurationField)
+class DurationField(Timedelta, BaseField):
+    """A field that handles days, hours, minutes and seconds
+    """
+    pass
```

Comment:
I guess the `pass` is superfluous

Reply:
Resolved with https://github.com/senaite/senaite.core/pull/2482/commits/f11328f18661de4d0c2b522267c95442930a7ecf